modules/web/webpack.config.js

```diff
@@ -85,7 +86,8 @@ var config = {
     },
     plugins: [
         extractCSSBundle,
-        extractThemes
+        extractThemes,
+        new UnusedFilesWebpackPlugin({pattern: '**/*.js'})
```

Comment:
*.* will complain about unused images and css too right? can we guarantee there is no css or svg which is not included using webpack aware api